src/sqlalchemy_cratedb/type/binary.py

```diff
@@ -0,0 +1,44 @@
+import base64
+import sqlalchemy as sa
+
+
+class LargeBinary(sa.String):
+
+    """A type for large binary byte data.
+
+    The :class:`.LargeBinary` type corresponds to a large and/or unlengthed
+    binary type for the target platform, such as BLOB on MySQL and BYTEA for
+    PostgreSQL.  It also handles the necessary conversions for the DBAPI.
+
+    """
+
+    __visit_name__ = "large_binary"
+
+    def bind_processor(self, dialect):
+        if dialect.dbapi is None:
+            return None
+
+        # TODO: DBAPIBinary = dialect.dbapi.Binary
+
+        def process(value):
+            if value is not None:
+                # TODO: return DBAPIBinary(value)
+                return base64.b64encode(value).decode()
+            else:
+                return None
+
+        return process
```

Comment:
_Note to self, or others who want to pick this up:_

Review that detail about returning `DBAPIBinary` or not.